cloudhub/modules/ROOT/pages/app-migration.adoc

```diff
@@ -46,6 +46,13 @@ If you're using `anypoint.dns.net` without a TLS context, the *Ingress* tab does
 .. Select whether you want to provide internal or external access to your endpoint.
 . Follow the steps to xref:cloudhub-2::ch2-deploy.adoc[deploy an application to CloudHub 2.0].
 
+[WARNING]
+====
+If you use a different name for the CloudHub 2.0 application, make sure you input the CloudHub application name in the subdomain field of the endpoint configuration within the *Ingress* tab settings. Failing to configure the correct URLs can result in traffic not reaching the CloudHub 2.0 application during xref:app-migration.adoc#next-steps-after-application-upgrade[traffic switching].
+
+By default, this action enables Dedicated Load Balancer (DLB) traffic switching. To enable Shared Load Balancer (SLB) traffic switching, add the CloudHub application endpoint in the format: `<app_name>.<region>.cloudhub.io`. For more information about SLB and DLB traffic switching, see xref:app-migration.adoc#understanding-the-traffic-source[Understanding the Traffic Source].
```

Comment:
Do you need "For more information about SLB and DLB traffic switching" or could you just say "See ..." (SF CX standard) Or at least, "For more information, see"

Reply:
I would leave it as is, because the linked section does no elaborate on enabling SLB traffic switching, but just provides theoretical info on SLB and DLB traffic source and traffic switching.

Reply:
Oh, gotcha. That makes sense then. Thanks for clarifying.